tox.ini

```diff
@@ -55,32 +66,56 @@ deps =
     Werkzeug==0.16.0
     zope.interface==4.6.0
 
-    {trial,coverage}: treq==18.6.0
-    {trial,coverage}: hypothesis==4.41.2
-    {trial,coverage}: idna==2.8
-    {trial,coverage}-py{27,py2}: mock==3.0.5
-    {trial,coverage}-py{27,py2}: typing==3.7.4.1
+    {test,coverage}: treq==18.6.0
+    {test,coverage}: hypothesis==4.41.2
+    {test,coverage}: idna==2.8
+    {test,coverage}-py{27,py2}: mock==3.0.5
+    {test,coverage}-py{27,py2}: typing==3.7.4.1
 
     coverage: coverage==4.5.4
 
-passenv =
-    # See https://github.com/codecov/codecov-python/blob/5b9d539a6a09bc84501b381b563956295478651a/README.md#using-tox
-    codecov: TOXENV
-    codecov: CI
-    codecov: TRAVIS TRAVIS_*
-
 setenv =
-    PIP_DISABLE_PIP_VERSION_CHECK=1
-    VIRTUALENV_NO_DOWNLOAD=1
```

Comment:
These aren't necessary any more.

Reply:
I'm open to that, but I wouldn't do that based on just these, since I think Tox still works (in most situations) if `pip` and `virtualenv` do these goofy things.